exercises/series/canonical-data.json

```diff
@@ -0,0 +1,134 @@
+{
+  "exercise": "series",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "slices of one from one",
+      "property": "slices",
+      "input": {
+        "series": "1",
+        "sliceLength": 1
+      },
+      "expected": ["1"]
+    },
+    {
+      "description": "slices of one from two",
+      "property": "slices",
+      "input": {
+        "series": "12",
+        "sliceLength": 1
+      },
+      "expected": ["1", "2"]
+    },
+    {
+      "description": "slices of two",
+      "property": "slices",
+      "input": {
+        "series": "12",
+        "sliceLength": 2
+      },
+      "expected": ["12"]
+    },
+    {
+      "description": "slices of two with nonsequential numbers",
+      "property": "slices",
+      "input": {
+        "series": "35",
+        "sliceLength": 2
+      },
+      "expected": ["35"]
+    },
+    {
+      "description": "slices of two overlap",
+      "property": "slices",
+      "input": {
+        "series": "9142",
+        "sliceLength": 2
+      },
+      "expected": ["91", "14", "42"]
+    },
+    {
+      "description": "slices can include duplicates",
+      "property": "slices",
+      "input": {
+        "series": "123123",
+        "sliceLength": 3
+      },
+      "expected": ["123", "231", "312", "123"]
+    },
+    {
+      "description": "strings can have repeating digits",
```

Comment:
Does this test case add anything to the "slices can include duplicates" test case?

Reply:
These two cases are essentially the same in my opinion.

Reply:
> Does this test case add anything to the "slices can include duplicates" test case?

It seemed to be duplicating efforts.  I combined `"slices can include duplicates"` and `"strings can have repeating digits"`.